Add missing arg to j_lock_buf_priv and j_unlock_buf_priv

mewmew opened the pull request.

mewmew commented:
I stumbled on this when looking at trying to make CaptureScreen in #225 bin exact. Didn't get that far, but at least found something :)

galaxyhaxz commented:
How did we miss this one? The arg is unused in 1.09 but is prevelant in the debug release. Seems to be for detecting "underflows":
```
void __fastcall j_dx_lock_mutex(BYTE idx)
{
    ++locktbl[idx];
    dx_lock_mutex();
}

void __fastcall j_dx_unlock_mutex(BYTE idx)
{
    if ( !locktbl[idx] )
        TermMsg("Draw lock underflow: 0x%x", idx);
    --locktbl[idx];
    dx_unlock_mutex();
}
```

mewmew commented:
> Does the debug build have sgdwLockCount?

Is `sgdwLockCount` located at `0x52A520` in 1.09b? If so, the debug release contains `sgdwLockCount` at address `0x69E44C`

mewmew commented:
The `locktbl` array is located at `0x69E460` in the debug release. I don't think this array is present in 1.09b.

mewmew commented:
> How did we miss this one? The arg is unused in 1.09 but is prevelant in the debug release. Seems to be for detecting "underflows":
>
> ```c
void __fastcall j_dx_lock_mutex(BYTE idx)
{
    ++locktbl[idx];
    dx_lock_mutex();
}

void __fastcall j_dx_unlock_mutex(BYTE idx)
{
    if ( !locktbl[idx] )
        TermMsg("Draw lock underflow: 0x%x", idx);
    --locktbl[idx];
    dx_unlock_mutex();
}
```

Thanks @galaxyhaxz! I added the debug definition to the PR. Also added a `debug` target to `MakefileVC`. I think this PR is ready for merge.

mewmew commented:
Merging, feel free to revert if I may have missed something.

mewmew commented:
Hmm, I did miss something. Accidentally pushed to a new branch at diasurgical rather than mewpull, so the old code was merged, rather than the updated one. Will fix.

mewmew commented:
> Hmm, I did miss something. Accidentally pushed to a new branch at diasurgical rather than mewpull, so the old code was merged, rather than the updated one. Will fix.

Fixed in 9772cd0743d729b44ed8e8ecb4afab49d2807ffc.
